src/test/java/com/hsbc/engineering/ArgumentTest.java

```diff
@@ -14,39 +14,42 @@ public class ArgumentTest {
     public static void setup() {
         arguments = new Arguments(new HashMap<>() {{
             put(Arguments.CLASS_NAME, Arguments.CLASS_NAME);
-            put(Arguments.USER,       Arguments.USER);
-            put(Arguments.PWD, Arguments.PWD);
-            put(Arguments.URL,        Arguments.URL);
-            put(Arguments.SQL,        Arguments.SQL);
-            put(Arguments.RUN_EXTENDED_TESTS, "true");
-        }});
+        put(Arguments.USER,       Arguments.USER);
+        put(Arguments.PWD, Arguments.PWD);
+        put(Arguments.URL,        Arguments.URL);
+        put(Arguments.SQL,        Arguments.SQL);
+        put(Arguments.RUN_EXTENDED_TESTS, "true");
+}});
```

Comment:
Maybe better use Map.of() here as well, because it's only once it's created, don't see any code that modifies it.